next.config.mjs

```diff
@@ -1,6 +1,14 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
         output: 'standalone',
+        async rewrites() {
+                return [
+                        {
+                                source: '/api/:path*',
+                                destination: 'http://api-dev.linchpino.com:8081/:path*',
```

Comment:
Can we use BASE_URL from system.js?

Reply:
> Can we use BASE_URL from system.js?

This is for test , remove line and push again